.github/workflows/positron-full-test.yml

```diff
@@ -103,7 +103,7 @@ jobs:
           curl https://raw.githubusercontent.com/posit-dev/qa-example-content/main/requirements.txt --output requirements.txt
           python -m pip install --upgrade pip
           python -m pip install -r requirements.txt
-          python -m pip install matplotlib ipykernel trcli
+          python -m pip install matplotlib ipykernel trcli graphviz
```

Comment:
On windows at least.. this also required an install of graphviz on my system, not just the python package. We might want to note that in the README

Reply:
And we need to add the installation of graphviz to positron-full-test.yml.
Looks like it might be pretty easy: https://github.com/marketplace/actions/setup-graphviz
```
- name: Setup Graphviz
  uses: ts-graphviz/setup-graphviz@v2.0.2
```

Reply:
Seems easy to add in!

Reply:
What's the reason for using graph-viz instead of just something like a standard scatter plot?

Reply:
I believe graph-viz is used to get a static plot.  We already had a dynamic plot test (scatter plot).